test/distributed/_shard/sharded_tensor/test_sharded_tensor_matrix_ops.py

```diff
@@ -0,0 +1,108 @@
+# Owner(s): ["oncall: distributed"]
+
+import sys
+
+import torch
+from torch.distributed._shard import (
+    sharded_tensor,
+)
+from torch.testing._internal.common_distributed import (
+    requires_nccl,
+    skip_if_lt_x_gpu,
+)
+from torch.testing._internal.common_utils import (
+    TEST_WITH_DEV_DBG_ASAN,
+    run_tests,
+)
+from torch.testing._internal.distributed._shard.sharded_tensor import (
+    ShardedTensorTestBase,
+    with_comms,
+)
+from torch.testing._internal.distributed._shard.sharded_tensor._test_st_common import (
+    _chunk_sharding_specs_list_for_test,
+)
+
+if TEST_WITH_DEV_DBG_ASAN:
+    print(
+        "Skip dev-asan as torch + multiprocessing spawn have known issues",
+        file=sys.stderr,
+    )
+    sys.exit(0)
+
+
+class TestShardedTernsorMatrixOps(ShardedTensorTestBase):
```

Comment:
Why do we have two similar classes and test files? There is `test_sharded_tensor_matrix_ops.py` and also `test_matrix_ops.py`?

Reply:
Oh because one is for the class methods of sharded tensor and another one is for the ops.. If we go with the proposal above, I think we can consolidate these two test files.